src/cmds/pbs_attach.c

```diff
@@ -397,6 +372,34 @@ main(int argc, char *argv[])
 		 */
 		(void)putenv("MPICH_PROCESS_GROUP=no");
 
+		/*
+		 ** Load in the job environment variables needed for cgroups
+		 ** Check to see the file $PBS_NODEFILE.env exists
```

Comment:
Well PBS_NODEFILE and jobid are the same value. I have changed the code to reference the jobid.